experiments/league.py

```diff
@@ -91,9 +91,7 @@ def parse_args():
     from ppo_gridnet import Agent, MicroRTSStatsRecorder
 
     from gym_microrts.envs.vec_env import MicroRTSBotVecEnv
-    from gym_microrts.envs.vec_env import (
-        MicroRTSGridModeSharedMemVecEnv as MicroRTSGridModeVecEnv,
-    )
+    from gym_microrts.envs.vec_env import MicroRTSGridModeVecEnv
```

Comment:
We will just use `MicroRTSGridModeVecEnv ` for the new migration for ease of maintenance